### PR TITLE
Simplify rule set

### PR DESCRIPTION
**Pull request by SpacePossum**

### CODE REVIEW

File: .php_cs.dist [hunk 1]
```diff
@@ -13,19 +13,17 @@ EOF;
...
         'no_extra_consecutive_blank_lines' => array('break', 'continue', 'extra', 'return', 'throw', 'use', 'parenthesis_brace_block', 'square_brace_block', 'curly_brace_block'),
         'no_useless_else' => true,
         'no_useless_return' => true,
         'ordered_class_elements' => true,
         'ordered_imports' => true,
-        'php_unit_construct' => true,
-        'php_unit_dedicate_assert' => true,
         'php_unit_strict' => true,
         'psr4' => true,
+        'silenced_deprecation_error' => false,
```
Comment (keradus) [inline .php_cs.dist]: actually we silence them as well, no ?

Comment (SpacePossum) [inline .php_cs.dist]: indeed we do, I searched in master.... ; )

**Comment by keradus** [comment]
So, now we are getting random hhvm failures ? new hhvm update on Travis env or what ?

**Comment by SpacePossum** [comment]
->
https://github.com/sebastianbergmann/object-enumerator/blob/1.0.0/src/Enumerator.php#L47
->
https://github.com/sebastianbergmann/recursion-context/blob/master/src/Context.php#L141

maybe that last ` $end[0]` access is a problem as it is not checked?

**Comment by keradus** [comment]
:+1:

File: src/RuleSet.php [hunk 2]
```diff
@@ -53,6 +53,7 @@
         ),
         '@Symfony' => array(
             '@PSR2' => true,
+            'array_syntax' => array('syntax' => 'long'),
```
Comment (keradus) [inline src/RuleSet.php]: I was thinking about it. Symfony keeps that rule only for framework project to not have merging hell with old branches that supported 5.3.3.

**Comment by keradus** [comment]
Thank you @SpacePossum.

**Comment by GrahamCampbell** [comment]
The symfony ruleset is still wrong.  Not all symfony projects do this. Only selected ones like the main framework.

**Comment by SpacePossum** [comment]
>The symfony ruleset is still wrong

wrong how? It was not changed in this PR

**Comment by GrahamCampbell** [comment]
Sorry, wrong PR. :P

**Pull request merged**